awx/ui_next/src/screens/Template/Templates.jsx

```diff
@@ -53,47 +56,49 @@ class Templates extends Component {
     return (
       <>
         <Breadcrumbs breadcrumbConfig={breadcrumbConfig} />
-        <PageSection>
```

Comment:
I think we either need to keep PageSection component at the top level, or use wrap all of the children components in PageSection.

Reply:
I think we need to add PageSection as close to the actual component that renders the view as possible.  I'm in favor of wrapping the children.